src/test/java/uk/ac/ebi/eva/pipeline/jobs/steps/VepInputGeneratorStepTest.java

```diff
@@ -45,37 +48,47 @@
  * Test {@link VepInputGeneratorStep}
  */
 @RunWith(SpringRunner.class)
-@TestPropertySource({"classpath:vep-input-generator-step.properties"})
+@TestPropertySource({"classpath:vep-input-generator-step.properties", "classpath:test-mongo.properties"})
 @ContextConfiguration(classes = {AnnotationJob.class, BatchTestConfiguration.class})
 public class VepInputGeneratorStepTest {
 
     private static final String MONGO_DUMP = "/dump/VariantStatsConfigurationTest_vl";
+
+    private static final String STUDY_ID = "7";
+
+    private static final String FILE_ID = "5";
+
     @Rule
     public TemporaryMongoRule mongoRule = new TemporaryMongoRule();
 
+    @Rule
+    public PipelineTemporaryFolderRule temporaryFolderRule = new PipelineTemporaryFolderRule();
+
     @Autowired
     private JobLauncherTestUtils jobLauncherTestUtils;
-    @Autowired
-    private JobOptions jobOptions;
-
-    @Before
-    public void setUp() throws Exception {
-        jobOptions.loadArgs();
-    }
 
     @Test
     public void shouldGenerateVepInput() throws Exception {
-        // TODO This test can't be changed to use temporary directory right now, as vepInput is a composite parameter
-        // that is not being recalculated at execution time.
-        mongoRule.restoreDump(getResourceUrl(MONGO_DUMP), jobOptions.getDbName());
-        File vepInputFile = new File(jobOptions.getVepInput());
+        String randomTemporaryDatabaseName = mongoRule.getRandomTemporaryDatabaseName();
+        mongoRule.restoreDump(getResourceUrl(MONGO_DUMP), randomTemporaryDatabaseName);
```

Comment:
there's a method in mongoRule `String restoreDumpInTemporaryDatabase(URL dumpLocation)`